targets/ubuntu-14.04-enterprise/Dockerfile

```diff
@@ -115,3 +97,7 @@ RUN echo $IMAGE_TAG > /opt/circleci/image_version
 ADD targets/ubuntu-14.04-enterprise/pkg-versions.sh /opt/circleci/bin/pkg-versions.sh
 
 LABEL circleci.user="ubuntu"
+
+USER ubuntu
+
+CMD ["/bin/bash"]
```

Comment:
I assume this is for when you want to run this container locally with `docker run` command? I used to do something like `docker run -it circleci/build-image:ubuntu-14.04-XXL-913-a78740d su - ubuntu` but I think `CMD` is better.